.travis.yml

```diff
@@ -16,20 +14,16 @@ matrix:
     env: TOXENV=mypy
   - python: 3.7
     env: TOXENV=docs
-
 install:
 - pip install -U tox codecov
-
 script: tox
-
 after_success:
 - codecov
-
 deploy:
   provider: pypi
-  user: kmike
+  user: scrapinghub
```

Comment:
@imduffy15 should it be `__token__` as well?

Reply:
To my surprise yes, pypi auto figures the user from the token.